packages/@aws-cdk/core/lib/cfn-parse.ts

```diff
@@ -107,6 +110,91 @@ export class FromCloudFormation {
     return ret;
   }
 
+  public static parseCreationPolicy(policy: any): CfnCreationPolicy | undefined {
```

Comment:
Add unit tests for all these methods. Even though they are not public outside of this package, the class is intended to be re-usable.

Reply:
Done.